Classes/Services/MetsExporter.php

```diff
@@ -793,6 +795,16 @@ public function getSlubData()
     }
 
     /**
+     * replaces single quote with double quote
+     * @param $string
+     * @return string
+     */
+    function replaceSingleQuote($string)
```

Comment:
While `private` would remove the helper function from the public interface making it simpler, what is the rational for having a `static` method?